src/main/java/mate/academy/Main.java

```diff
@@ -27,7 +39,8 @@ public static void main(String[] args) {
         secondCinemaHall.setCapacity(200);
         secondCinemaHall.setDescription("second hall with capacity 200");
 
-        CinemaHallService cinemaHallService = null;
+        CinemaHallService cinemaHallService
```

Comment:
The variable declaration for `CinemaHallService` is missing an `=` sign. It should be `CinemaHallService cinemaHallService = ...`.